packages/circuits/package.json

```diff
@@ -12,14 +12,14 @@
     "publishtonpm": "echo 'not published'",
     "test-publish": "echo 'not published'",
     "circom": "circom",
-    "build-cricuit": "cd scripts && sh build.sh build",
-    "dev-install": "/bin/bash ./scripts/build.sh install",
-    "dev-setup": "/bin/bash ./scripts/build.sh setup",
-    "dev-setup-test": "/bin/bash ./scripts/build.sh setup-test",
-    "gen-proof": "/bin/bash ./scripts/build.sh gen-proof",
-    "gen-witness": "/bin/bash ./scripts/build.sh gen-witness",
-    "gen-contracts": "/bin/bash ./scripts/build.sh gen-contracts",
-    "verif-proof": "/bin/bash ./scripts/build.sh verify-proof",
+    "build-circuit": "sh ./scripts/build.sh build",
```

Comment:
As it will change the flow of setting up the repo, can this be added as instructions in the README?

Reply:
Right. I have added it now.